src/components/ui/Tooltip.tsx

```diff
@@ -10,7 +10,7 @@ import classnames, {
   zIndex,
 } from 'classnames/tailwind'
 
-const tooltipClasses = classnames(
+const TooltipClasses = classnames(
```

Comment:
Why do you use `CamesCase` not `camelCase` here?

Reply:
it's a mistake. fixed it, ty